### PR TITLE
Fix my source files

### PR DESCRIPTION
**Pull request by TarasMavrovych**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+.img{
+  width: 180px;
+  height: 180px;
+
+  border-radius: 50%;
+
+  /* outline: 1px solid #000;
+  border: 3px solid #cacaca; */
```
Comment (PavloSukhinin) [inline assets/css/style.css]: keep your code clean, don't leave commented code

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
+body{
```
Comment (PavloSukhinin) [inline assets/css/style.css]:
```suggestion
body {
```

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  background-color: #f7f7f7;
+}
+
+.main_section{
```
Comment (PavloSukhinin) [inline assets/css/style.css]:
```suggestion
.main_section {
```

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  display: flex;
+  margin: 0 auto;
+
+  width: 810px;
+
+  background: white;
+}
+
+.my_sidebar{
```
Comment (PavloSukhinin) [inline assets/css/style.css]: apply this for all classes
```suggestion
.my_sidebar {
```

File: index.html [hunk 5]
```diff
@@ -5,11 +5,42 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-    <link rel="stylesheet" href="/teras-mavr-cv/assets/css/reset.css">
-
     <title>Document</title>
+
+    <link rel="stylesheet" href="../teras-mavr-cv/assets/css/reset.css">
+    <link rel="stylesheet" href="../teras-mavr-cv/assets/css/style.css">
```
Comment (PavloSukhinin) [inline index.html]:
```suggestion
    <link rel="stylesheet" href="assets/css/reset.css">
    <link rel="stylesheet" href="assets/css/style.css">
```
this is how your code structured

File: index.html [hunk 6]
```diff
@@ -5,11 +5,42 @@
...
 </head>
 <body>
+  <!-- main_section -->
+  <div class="main_section">
+
+    <!--my sidebar-->
+    <aside class="my_sidebar">
+
+      <div class="my_img">
+        <img class="img" src="../teras-mavr-cv/self_photo.jpg" alt="photo">
```
Comment (PavloSukhinin) [inline index.html]:
```suggestion
        <img class="img" src="assets/img/self_photo.jpg" alt="photo">
```
keep your files structured

File: index.html [hunk 7]
```diff
@@ -5,11 +5,42 @@
...
     <title>Document</title>
+
+    <link rel="stylesheet" href="../teras-mavr-cv/assets/css/reset.css">
+    <link rel="stylesheet" href="../teras-mavr-cv/assets/css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;400;500;700&display=swap" rel="stylesheet">
+
 </head>
 <body>
+  <!-- main_section -->
+  <div class="main_section">
```
Comment (PavloSukhinin) [inline index.html]: [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) why not ```main```?

**Review by PavloSukhinin — COMMENTED** [review]
you should give meaningful names to your commits ```for my task 1``` doesn't make much sense

**Review by vinnikukr — COMMENTED** [review]
Two photos? Delete redundant from root folder

**Pull request closed without merging**